exampleSite/config.toml

```diff
@@ -15,26 +15,26 @@ googleAnalytics = ""
 
   # Branding options
   [params.branding]
-    logoNavigation = "logo.svg"
-    textNavigation = "Syna"
+    logo_navigation = "logo.svg"
+    text_navigation = "Syna"
 
   [params.style]
     #background = "secondary"
-    #backgroundNavigation = "light" # white, light, secondary, primary, dark - default: dark
-    #backgroundFooter = "light" # white, light, secondary, primary, dark - default: dark
+    #background_navigation = "light" # white, light, secondary, primary, dark - default: dark
+    #background_footer = "light" # white, light, secondary, primary, dark - default: dark
```

Comment:
background_footer isn't used anymore correct? As we moved the copyright into it a fragment.

Reply:
Fixed in 02991898802d2faa7c8be867619a655643a2edb9

Reply:
Fixed in 02991898802d2faa7c8be867619a655643a2edb9